packages/types/src/index.ts

```diff
@@ -13,6 +13,7 @@ export { Options } from './options';
 export { Package } from './package';
 export { Request } from './request';
 export { Response } from './response';
+export { Runtime } from './runtime';
```

Comment:
Hmm, forgot to commit a file `runtime.ts`?

Builds are failing in this line => https://github.com/getsentry/sentry-javascript/pull/2513#issuecomment-603865629